android/res/layout/fragment_phone.xml

```diff
@@ -25,7 +25,7 @@
     <TextView
         android:id="@+id/tv__append_phone"
         android:layout_width="match_parent"
-        android:layout_height="@dimen/height_block_base"
```

Comment:
Is this identifier used somewhere else? If not, please clean it up.

Reply:
This is also used in a lot of other places
![image](https://user-images.githubusercontent.com/46300244/230200469-4ad06f91-919d-42ed-8cad-7bec83b60af5.png)